tests/Integration/ProxyEndpointTest.php

```diff
@@ -181,22 +185,13 @@ private function run_test_get_items_fails(
 		array $options,
 		string $expected_error_code,
 		string $expected_error_message,
-		$request = null
+		?WP_REST_Request $request = null
 	): void {
 		TestCase::set_options( $options );
 		if ( null === $request ) {
 			$request = new WP_REST_Request( 'GET', self::$route );
 		}
 
-		$dispatched = 0;
-		add_filter(
-			'pre_http_request',
-			function () use ( &$dispatched ) {
-				$dispatched++;
-				return null;
-			}
-		);
```

Comment:
This is the code that was removed. As far as I can understand, it was never fired as in the specific context, an HTTP request was never fired. We were also not testing the `dispatched` value, thing which we're doing in other tests which actually perform an HTTP request.